src/player.cpp

```diff
@@ -12119,51 +12119,26 @@ int player::encumb( body_part bp ) const
  */
 int player::encumb(body_part bp, double &layers, int &armorenc) const
 {
-    int ret = 0;
-    double layer[MAX_CLOTHING_LAYER] = { };
-    int level = 0;
-    bool is_wearing_active_power_armor = false;
-    for( auto &w : worn ) {
-        if( w.active && w.is_power_armor() ) {
-            is_wearing_active_power_armor = true;
-            break;
-        }
-    }
+    int layer[MAX_CLOTHING_LAYER] = { };
 
-    for( auto& w : worn ) {
-        if( !w.covers(bp) ) {
-            continue;
-        }
+    for (auto w = worn.begin(); w != worn.end(); ++w) {
+        if (! w->covers(bp)) continue;
 
-        if( w.has_flag( "SKINTIGHT" ) ) {
-            level = UNDERWEAR;
-        } else if ( w.has_flag( "WAIST" ) ) {
-            level = WAIST_LAYER;
-        } else if ( w.has_flag( "OUTER" ) ) {
-            level = OUTER_LAYER;
-        } else if ( w.has_flag( "BELTED") ) {
-            level = BELTED_LAYER;
-        } else {
-            level = REGULAR_LAYER;
-        }
+        if (! (w->has_flag("FIT") && w->get_encumber() == 0))
+            layer[w->get_layer()] += 10;
```

Comment:
This was intentionally removed, we'd need a good rationale to put it back in, see #12741 for the most recent discussion on the subject, which is leaning toward having the layering penalty scale with underlying encumbrance rather than negating it altogether.